yt_dlp/extractor/altcensored.py

```diff
@@ -32,7 +32,7 @@ class AltCensoredIE(InfoExtractor):
             'duration': 926.09,
             'thumbnail': 'https://archive.org/download/youtube-k0srjLSkga8/youtube-k0srjLSkga8.thumbs/k0srjLSkga8_000925.jpg',
             'view_count': int,
-            'categories': ['News & Politics'],
+            'categories': ['News', '&', 'Politics'],
```

Comment:
this is wrong and the cause of it should be fixed in a separate PR/commit
```suggestion
            'categories': ['News & Politics'],  # FIXME
```